.github/workflows/_test_pax.yaml

```diff
@@ -65,7 +70,7 @@ jobs:
           NODES=$(((TOTAL_TASKS+MAX_GPUS_PER_NODE-1)/MAX_GPUS_PER_NODE))
           GPUS_PER_NODE=$((TOTAL_TASKS/NODES))
 
-          JOB_NAME=${GITHUB_RUN_ID}-${TEST_CASE_NAME}
+          JOB_NAME=${GITHUB_RUN_ID}-${TEST_CASE_NAME}${{ inputs.SUFFIX }}
```

Comment:
Would `GITHUB_RUN_ID` + `TEST_CASE_NAME` + `GITHUB_RUN_NUMBER` suffice to prevent overwriting of artifacts in the CI cluster?

Or is it the case that we are running the same set of MGMN tests but just plugging in either the upstream or Rosetta pax container? In this case, the `ARTIFACT_NAME` convention as above might be a more generic solution.

Reply:
Yes, we run some of the same tests with both the upstream and rosetta containers. I might prefer `ARTIFACT_NAME `  in this case as it is more explicit about which version of the test is being run.